app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
+
+  def recent_posts
+    posts.limit(3).order(created_at: :desc)
+  end
```

Comment:
- [ ] In this method your data should be ordered in descending order before you get the first three rows of data :thinking: so your method should look something like below :arrow_down: :blush:
```ruby
  def recent_posts
    posts.limit(3).order(created_at: :desc)
  end
```